src/native/xamarin-debug-app-helper/debug-app-helper.cc

```diff
@@ -234,9 +238,10 @@ get_libmonosgen_path ()
 		log_fatal (LOG_DEFAULT, "  %s", app_lib_dir);
 	}
 
-	log_fatal (LOG_DEFAULT, "Do you have a shared runtime build of your app with AndroidManifest.xml android:minSdkVersion < 10 while running on a 64-bit Android 5.0 target? This combination is not supported.");
-	log_fatal (LOG_DEFAULT, "Please either set android:minSdkVersion >= 10 or use a build without the shared runtime (like default Release configuration).");
-	Helpers::abort_application ();
+	Helpers::abort_application (
+		"Do you have a shared runtime build of your app with AndroidManifest.xml android:minSdkVersion < 10 while running on a 64-bit Android 5.0 target? This combination is not supported. "
```

Comment:
If the maximum assertion message length is 120 characters, doesn't this message -- at 305 characters -- vastly exceed it?

Reply:
It does, but the message is logged separately, too, so we'll get the full message in logcat.

Reply:
This particular message might no longer be needed, since we don't support API10 anymore.